### PR TITLE
[AzureMonitorExporter] Update Changelog for _OTELRESOURCE_ metrics

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Follow up: #36063

### CODE REVIEW

**Review by TimothyMothra — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+* Added resource attributes to metric telemetry with a name of `_OTELRESOURCE_`, which includes all attributes as custom dimensions in every transmission to the ingestion service, enabling the storage of resource objects in the `AppResources` table. ([36063](https://github.com/Azure/azure-sdk-for-net/pull/36063))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]:
```suggestion
* [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
attributes will now be exported as custom dimensions on CustomMetric telemetry
with name as `_OTELRESOURCE_`. This CustomMetric will be included in every batch
of telemetry items sent to ingestion, enabling the storage of resource objects
in the `AppResources` table in kusto logs.
([36063](https://github.com/Azure/azure-sdk-for-net/pull/36063))
```

**Review by vishweshbankwar — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+* [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) attributes will now be exported as custom dimensions on Metric telemetry with the name `_OTELRESOURCE_`. This Metric will be included in every batch of telemetry items sent to the ingestion service. ([#36063](https://github.com/Azure/azure-sdk-for-net/pull/36063))
```
[the remark this answers is not in the document]
Comment (rajkumar-rangaraj) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: Two copies of `_OTELRESOURCE_` will be sent to ingestion service, one with traces batch and another with metrics batch.

[the remark this answers is not in the document]
Comment (reyang) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: Which means the resulting metrics could have totally different dimensions? (what should the user expect from this metrics stream?)

[the remark this answers is not in the document]
Comment (reyang) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: Got it, thanks.

**Pull request merged**